influxdb_store.go

```diff
@@ -56,6 +59,12 @@ type InfluxDBStore struct {
 	// When set to `testMode` - `testDBName` will be dropped and created, so newly database is ready for tests.
 	server        *influxDBServer.Server // InfluxDB API server.
 	tracesPerPage int                    // Number of traces per page.
+
+	batchMu         sync.Mutex
+	batchSizeBytes  int
+	batch           []influxDBClient.Point
+	flusherStopChan chan struct{}
+	log             *log.Logger
```

Comment:
is there a way to ensure this is always initialized to log to devnull? That way you don't need to do the `if log != nil` pattern.